api/v1/controllers/RegistrationController.js

```diff
@@ -377,6 +396,9 @@ router.put('/attendee/decision/:id',  middleware.request(requests.AttendeeDecisi
 router.put('/attendee/:id(\\d+)', middleware.request(requests.AttendeeRequest),
 	middleware.permission(roles.ORGANIZERS), updateAttendeeById);
 
+router.put('/sendlist', middleware.request(requests.SendListRequest), 
```

Comment:
If it is, then we should probably put it in a different controller (since the lists can apply to more than just registration). Maybe a `MailController`. In that case `/send` would be appropriate.